internal/cli/organizations.go

```diff
@@ -422,6 +424,59 @@ func openOrganizationCmd(cli *cli) *cobra.Command {
 	return cmd
 }
 
+func membersOrganizationCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		ID     string
+		Number int
+	}
+
+	cmd := &cobra.Command{
+		Use:   "members",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "List members of an organization",
+		Long:  "List members of an organization.",
+		Example: `auth0 orgs members 
+auth0 orgs members <id>`,
```

Comment:
This command should be `auth0 orgs member list <id>`. See https://auth0.github.io/auth0-cli/auth0_apis_scopes_list.html for a similar example (`auth0 apis scopes list <id>`).